### ISSUE TITLE
Loadscreens: Add structure placement and factory duplication

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K#2885]
- [ ] There should be a loadscreen with a big list of all the structure placement modifiers. Include skydust. People won't read everything on the first viewing but they will see it many times.
- [ ] Add a loadscreen with basic factory tips. Your first factory is free. Treat them like tech unlocks as they are expensive. The best way to increase production is with nano turrets.

**Comment by The-Yak** [comment]
post-release?

**Comment by sprunk** [comment]
Not necessarily, the earlier the better. Loadscreens aren't a new feature so the feature freeze doesn't apply.

**Comment by The-Yak** [comment]
referencing https://github.com/ZeroK-RTS/Zero-K/issues/2730

**Comment by sprunk** [comment]
Skydust means that the loadscreen should explain the methods used to elevate a building:
 * hold LMB to toggle height mode, then mouse up/down to change height.
 * press B to toggle height mode, then C/V to go change height.
 * hold ALT and scroll the mousewheel to change height (automatically enables height mode). Hold ALT and press MMB to toggle height mode.

**Comment by sprunk** [comment]
>Can't reproduce

Maybe you have it disabled? Should be on by default https://github.com/ZeroK-RTS/Zero-K/blob/master/LuaUI/Widgets/gui_lasso_terraform.lua#L90

**Comment by The-Yak** [comment]
Got it now, had to hold left click for longer then I anticipated. Alt-mmb is still my preferred method.

**Comment by The-Yak** [comment]
I don't like the tech unlocks comparison, it will only make sense to experienced RTS gamers. I think it can be dumbed down to "don't make multiple facs".

First draft of factory loadscreen:
![factoryscreen](https://user-images.githubusercontent.com/1368663/38525594-2a6c11ae-3c10-11e8-8fbd-8b3438e44067.jpg)
https://i.imgur.com/cjyurMy.jpg

**Comment by GoogleFrog** [comment]
It is a bit direct and negative as well as being a block of text. Perhaps a list of what each BP source is good at would be better, with the factory tip nearby.

**Comment by The-Yak** [comment]
Each BP source? Do you mean metal and energy? There's already a loadscreen for those.

**Comment by The-Yak** [comment]
>Build factories to unlock sets of units. The first one is free.

This is not true in the context of the campaign, unit unlocks require more then just building the right factory.

>Build nanotowers to assist factory construction efficiently.

This is an improvement on my wording, perhaps "or guard with constructors" could be added like anarchid said.

>Build constructors to expand your territory.

This one is debatably not good advise. Constructors are multirole units and only some of them are good at expansion (welder) , and commander is arguably better for expansion. 

>I would spend two words more and say that the first factory is free and instant.

Why? I don't think it's helpful at all, instant factory is obvious after first real game and doesn't need to be communicated.

I will be image dumping what I have got so far on the forums soon.

**Comment by GoogleFrog** [comment]
Don't put a panel around the logo.

**Comment by The-Yak** [comment]
Alright

**Comment by The-Yak** [comment]
Useful shortcuts refresh with new logo

![usefulshortcuts](https://user-images.githubusercontent.com/1368663/38754788-629074fe-3f20-11e8-9d5b-0bcf5590b036.jpg)

**Comment by Lasica** [comment]
Cool, I like this one. Not sure the "gather" is the right word, but I can't think of anything better.  Manage, order, maybe just "select"? It could say about standard unit groups ctrl + number, and number pressing to select them back again and auto group which is now default - alt + number. That would be a lot of text on one screen though, maybe it can be divided on 2 screens
